docs/install/install-redisinsight/install-on-k8s.md

```diff
@@ -0,0 +1,314 @@
+spec:
+  replicas: 1 #a single replica pod
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: redisinsight #which pods is the deployment managing, as defined by the pod template
+  template: #pod template
+    metadata:
+      labels:
+        app: redisinsight #label for pod/s
+    spec:
+      volumes:
+        - name: db
+          persistentVolumeClaim:
+            claimName: redisinsight-pv-claim
+      initContainers:
+        - name: init
+          image: busybox
+          command:
+            - /bin/sh
+            - '-c'
+            - |
+              chown -R 1001 /db
+          resources: {}
+          volumeMounts:
+            - name: db
+              mountPath: /db
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+      containers:
+        - name:  redisinsight #Container name (DNS_LABEL, unique)
+          image: redis/redisinsight:latest #repo/image
+          imagePullPolicy: IfNotPresent #Always pull image
+          volumeMounts:
+          - name: db #Pod volumes to mount into the container's filesystem. Cannot be updated.
+            mountPath: /db
+          ports:
+          - containerPort: 5540 #exposed container port and protocol
+            protocol: TCP
+```
+
+2. Create the RedisInsight deployment and service.
+
+```sh
+kubectl apply -f redisinsight.yaml
+```
+
+## Create the RedisInsight deployment without a service.
+
+Below is an annotated YAML file that will create a RedisInsight
+deployment in a K8s cluster.
+
+1. Create a new file redisinsight.yaml with the content below
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redisinsight #deployment name
+  labels:
+    app: redisinsight #deployment label
+spec:
+  replicas: 1 #a single replica pod
+  selector:
+    matchLabels:
+      app: redisinsight #which pods is the deployment managing, as defined by the pod template
+  template: #pod template
+    metadata:
+      labels:
+        app: redisinsight #label for pod/s
+    spec:
+      containers:
+      - name:  redisinsight #Container name (DNS_LABEL, unique)
+        image: redis/redisinsight:latest #repo/image
+        imagePullPolicy: IfNotPresent #Always pull image
+        env:
+          # If there's a service named 'redisinsight' that exposes the
+          # deployment, we manually set `REDISINSIGHT_HOST` and
+          # `REDISINSIGHT_PORT` to override the service environment
+          # variables.
+          - name: REDISINSIGHT_HOST
+            value: "0.0.0.0"
+          - name: REDISINSIGHT_PORT
+            value: "5540"
+        volumeMounts:
+        - name: db #Pod volumes to mount into the container's filesystem. Cannot be updated.
+          mountPath: /db
+        ports:
+        - containerPort: 5540 #exposed conainer port and protocol
+          protocol: TCP
+        livenessProbe:
+           httpGet:
+              path : /healthcheck/ # exposed RI endpoint for healthcheck
```

Comment:
There is no healcheck endpoint
Let's implement it

Reply:
let's discuss